scripts/uploader.rb

```diff
@@ -114,6 +138,11 @@ def uploaded_filename(appliance_name)
       File.basename(filename)
     end
 
+    def devel_filename(appliance_name)
+      name = appliance_name.split("-")
+      (name[0..1] << "devel").join("-") + File.extname(appliance_name)
```

Comment:
Our GCE image has `.tar.gz` extension `manageiq-gce-master-<date>-<sha>.tar.gz`, so use of `extname` will change to `.gz` and GCE upload won't work.